plasmapy/physics/parameters.py

```diff
@@ -713,32 +716,42 @@ def collision_rate_ion_ion(T_i, n_i, ion_particle,
     <Quantity 95918.76240877 1 / s>
 
     """
-    from plasmapy.physics.transport.collisions import Coulomb_logarithm
+    from plasmapy.physics.transport.collisions import (Coulomb_logarithm,
+                                                       collision_frequency)
     T_i = T_i.to(u.K, equivalencies=u.temperature_energy())
     m_i = atomic.ion_mass(ion_particle)
-    if V is not None:
-        V = V
+    particles = [ion_particle, ion_particle]
+    if V:
+        pass
     else:
```

Comment:
You could just do `if not V: V = np.sqrt(...)`, no?

Reply:
Actually no, because we have to handle zero V via  `if V is not None`.

Reply:
Shouldn't we be raising an error when `V=0`? This basically means the user messed up somewhere and is passing us a static system where collisions are not possible.

Reply:
Okay, you're right. I completely missed that.

Reply:
On second thought, I'm not sure if we should raise an exception if `V=0`, since there may be instances where people are plotting collision_rate stuff as a function of `V`, and would use `V=0` as an endpoint in the scan.  I imagine this use would be pretty rare though.

Reply:
@namurphy What would you return though? I don't think you can return a sensible collision frequency for a static system.

Reply:
Zero collisions per unit time (any time, in fact) seems reasonable enough!

Reply:
In fact, barring errors and just looking at the math of it, that seems to be exactly what we're returning right now, given that frequency is ~V^3.

Reply:
Please disregard what I said. Mistakes were made.